packages/ui-components/src/components/PasswordStrengthBar.js

```diff
@@ -0,0 +1,81 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Typography from '@material-ui/core/Typography';
+import MuiBox from '@material-ui/core/Box';
+import PasswordStrengthBarComponent from 'react-password-strength-bar';
+
+const Label = styled(Typography)`
+  font-size: 0.9rem;
+  line-height: 1.2rem;
+  color: ${props => props.theme.palette.text.primary};
+  margin-bottom: 0.6rem;
+`;
+
+const HelperText = styled(Typography)`
+  font-size: 0.9rem;
+  line-height: 1rem;
+  color: ${props => props.theme.palette.text.tertiary};
+  margin-top: 1rem;
+`;
+
+export const PasswordStrengthBarFallback = styled.div`
```

Comment:
So this is where you could have something that looked like a fuzzy version of a strength bar, if you wanted to make it fancy?

Reply:
yep it's basically just a loader. Obviously it doesn't need to be here - it could be made directly in the suspense component but I wanted to make it easy for future implementations.